src/util.js

```diff
@@ -74,18 +74,25 @@ util.getAttrs = function( tagMarkup, settings )
     }
 };
 
-util.getRemote = function( uri, callback, toDataUri )
+function getRemote( uri, settings, callback, toDataUri )
 {
     if( /^\/\//.test( uri ) )
     {
         uri = "https:" + uri;
     }
 
+    var requestOptions = {
+        uri: uri,
+        encoding: toDataUri ? "binary" : ""
+    };
+
+    if( typeof settings.requestTransform === "function" )
+    {
+        requestOptions = settings.requestTransform( requestOptions ) || requestOptions;
```

Comment:
There is no such thing as falsey object in JS. All the objects always are truthy. I added this because I started to use this option myself already and found out that I kept forgetting to return `requestOptions` back. As for cancelling request, interesting idea. We could cancel it when the function explicitly returns `false`.